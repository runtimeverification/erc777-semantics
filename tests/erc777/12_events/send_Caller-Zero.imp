test.preamble();

// Testing send(To, Value, "") when To = caller and Value = 0.

test.send(caller, 0, "");

printLogs;
