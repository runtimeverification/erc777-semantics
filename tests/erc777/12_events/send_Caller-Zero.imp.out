7 is sending 0 to 7 (data:)
Balance of 7 is 70
Balance of 7 is 70

Events:
Minted(8, 10, 100, , init)
Transfer(0, 10, 100)
Minted(8, 9, 90, , init)
Transfer(0, 9, 90)
Minted(8, 8, 80, , init)
Transfer(0, 8, 80)
Minted(8, 7, 70, , init)
Transfer(0, 7, 70)
Minted(8, 6, 60, , init)
Transfer(0, 6, 60)
Minted(8, 5, 50, , init)
Transfer(0, 5, 50)
Minted(8, 4, 40, , init)
Transfer(0, 4, 40)
Minted(8, 3, 30, , init)
Transfer(0, 3, 30)
Minted(8, 2, 20, , init)
Transfer(0, 2, 20)
Minted(8, 1, 10, , init)
Transfer(0, 1, 10)
Sent(7, 7, 7, 0, , )
Transfer(7, 7, 0)
